src/main/resources/application.yaml

```diff
@@ -402,7 +402,7 @@ docmosis:
       REJECTED_REFUSAL_ORDER_COVER_LETTER: FL-NFD-GOR-ENG-Conditional-Order-Rejected-Refusal-Cover-Letter_V2.docx #TODO: update welsh content
       CLARIFICATION_REFUSAL_ORDER: FL-NFD-GOR-WEL-Conditional-Order-Clarification-Refusal-Order.docx
       CLARIFICATION_REFUSAL_ORDER_COVER_LETTER: FL-NFD-GOR-ENG-Conditional-Order-Clarification-Refusal-Cover-Letter_V2.docx #TODO: update welsh content
-      JUDICIAL_SEPARATION_CONDITIONAL_ORDER_REFUSAL_COVER_LETTER: FL-NFD-GOR-ENG-Judicial-Separation-Conditional-Order-Refusal-Cover-Letter_V1.docx #TODO: update welsh content
+      JUDICIAL_SEPARATION_CONDITIONAL_ORDER_REFUSAL_COVER_LETTER: FL-NFD-GOR-WEL-Judicial-Separation-Conditional-Order-Refusal-Cover-Letter_V1.docx #TODO: update welsh content
```

Comment:
Minor comment - can we remove the TODO comment as this has now been done.

Reply:
Done!